atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CqlExecutorImpl.java

```diff
@@ -71,12 +74,18 @@
             TableReference tableRef,
             byte[] startRowInclusive,
             int limit) {
+        String selQuery = "SELECT key, column1, column2 FROM %s WHERE token(key) >= token(%s) LIMIT %s;";
```

Comment:
would it be better to pass the `CqlQuery` object into the thrift client and trace it there? currently we are splitting the tracing between two classes.

Reply:
Done.